airflow/providers/microsoft/azure/operators/azure_container_instances.py

```diff
@@ -136,16 +136,16 @@ def __init__(
         name: str,
         image: str,
         region: str,
-        environment_variables: Optional[Dict[Any, Any]] = None,
+        environment_variables: Optional[dict] = None,
         secured_variables: Optional[str] = None,
-        volumes: Optional[List[Any]] = None,
+        volumes: Optional[list] = None,
         memory_in_gb: Optional[Any] = None,
         cpu: Optional[Any] = None,
         gpu: Optional[Any] = None,
         command: Optional[str] = None,
         remove_on_error: bool = True,
         fail_if_exists: bool = True,
-        tags: Optional[Dict[str, str]] = None,
+        tags: Optional[dict] = None,
```

Comment:
```suggestion
        tags: Optional[Dict[str, str]] = None,
```

isn't `Dict[str, str] more appropriate here?

Reply:
it is, but I thought with python 3.9 we can have dict[str, str], hence kept it as dict only. Maybe I am thinking in future! :)

Reply:
Haha man we still support Py 2.7 for Airflow 1.10.x, imagine :D